doc/Comments.xml

```diff
@@ -297,6 +319,25 @@ Order(S5);
 ]]></Listing>
 </Subsection>
 
+
+<Subsection Label="@Listing">
```

Comment:
I looked at the GAPDoc documentation, and it offers listings, examples, and logs. We should *not* confuses people more by calling GAPDoc examples "listings". Let's have @Listing produce a GAPDoc List, and lts' have `@Log` begin a GAPDoc log. And if we can't have @Example begin a GAPDoc example (sad :-/), then we pick a different name for that. Don't have a bright idea for that right now, though... hrm